nsv13/code/modules/overmap/ai-skynet.dm

```diff
@@ -707,15 +717,26 @@ Adding tasks is easy! Just define a datum for it.
 	taunts = list("Better crews have tried to cross the Rubicon, you will die like they did.", "Defense force, stand ready!", "Nanotrasen filth. Munitions, ready the guns. We’ll scrub the galaxy clean of you vermin.", "This shift just gets better and better. I’ll have your Captain’s head on my wall.")
 	fleet_trait = FLEET_TRAIT_DEFENSE
 
-/datum/fleet/earthbuster
+/datum/fleet/earthbuster //Depricated
 	name = "\proper Syndicate Armada" //Fleet spawned if the players are too inactive. Set course...FOR EARTH.
 	destroyer_types = list(/obj/structure/overmap/syndicate/ai, /obj/structure/overmap/syndicate/ai/nuclear, /obj/structure/overmap/syndicate/ai/assault_cruiser, /obj/structure/overmap/syndicate/ai/gunboat, /obj/structure/overmap/syndicate/ai/submarine, /obj/structure/overmap/syndicate/ai/assault_cruiser/boarding_frigate)
 	size = FLEET_DIFFICULTY_VERY_HARD
 	allow_difficulty_scaling = FALSE
 	taunts = list("We're coming for Sol, and you can't stop us. All batteries fire at will.", "Lay down your arms now, you're outnumbered.", "All hands, assume assault formation. Begin bombardment.")
 	audio_cues = list()
 
-/datum/fleet/interdiction	//Pretty strong fleet with unerring hunting senses, Adminspawn for now.
+/datum/fleet/armada
+	name = "\proper Syndicate Armada" //Fleet used in the armada gamemode for the final confrontation
```

Comment:
Names starting with a capital letter a proper by default

Reply:
Also are you sure you want it to be proper? 
proper name reference = "It's Syndicate Armada"
improper name reference = "It's the Syndicate Armada"

Generally you only want it for pure names
```suggestion
	name = "\improper Syndicate Armada" //Fleet used in the armada gamemode for the final confrontation
```